docs/processors/builtin/unwrap.debezium.mdx

```diff
@@ -11,7 +11,7 @@ import Box from "@mui/system/Box";
 
 # `unwrap.debezium`
 
-Unwraps a Debezium record from the input OpenCDC record.
+Unwraps a Debezium record from the input [OpenCDC record](/docs/features/opencdc-record).
```

Comment:
We'll want to change this in the code, i.e. https://github.com/ConduitIO/conduit/blob/main/pkg/plugin/processor/builtin/impl/unwrap/debezium.go#L67

Reply:
@hariso you're right. This is autogenerated. I'll take care of that.

Reply:
@hariso opened this other pull-request to update the source code https://github.com/ConduitIO/conduit/pull/1428.